### ISSUE TITLE
Dev: Makefile

### ISSUE THREAD

**Issue by JoeKae** [YarostheLaunchpadder/MissionPlanner-to-Litchi#18]
We should bundle processes for testing, version increments, building etc. by using a Makefile.

**Comment by YarostheLaunchpadder** [comment]
What actions would be done in the makefile? Downloading the neccesasry packages, compiling the code?
Please give more information about this.
Thanks :)

**Comment by JoeKae** [comment]
Yes for example the build process would be a Makefile task. You can view the Makefile as an alias mapping where you can trigger long complicated commands with a short make command. Also you can trigger a whole pipeline of make commands with a single one.
